### DISCUSSION TITLE
I cant Properly trade with villagers

### DISCUSSION THREAD

**Discussion by Chief0609** [FabricMC/fabric-api#2855] [Mod Dev Support]
Villagers/Entities with trades wont allow me to trade properly
[Need Help]
i have been playing fabric 1.19.2 for awhile now and every time i try to trade with any entity i cant pick what i want to trade or shift click or do anything, has anyone else gotten this problem? I can still access the trade menu and can trade if i meet the proper requirements but if something has the same price and i want to switch to that im S.O.L. Ive tried making a completely different modpack, without changing any of the settings and still had the same issue, im currently trying to use the process of elimination to see what mod maybe be causing this but i have no information apart from that

the list of mods that im using:

https://www.curseforge.com/minecraft/mc-mods/fabric-disable-custom-worlds-advice
https://www.curseforge.com/minecraft/mc-mods/accurate-block-placement
https://www.curseforge.com/minecraft/mc-mods/advanced-nbt-tooltips
https://www.curseforge.com/minecraft/mc-mods/aim-assistance-fabric
https://www.curseforge.com/minecraft/mc-mods/anti-tool-break-fabric
https://www.curseforge.com/minecraft/mc-mods/appleskin
https://www.curseforge.com/minecraft/mc-mods/architectury-api
https://www.curseforge.com/minecraft/mc-mods/armor-hotswap
https://www.curseforge.com/minecraft/mc-mods/arrp
https://www.curseforge.com/minecraft/mc-mods/badpackets
https://www.curseforge.com/minecraft/mc-mods/better-enchanted-books
https://www.curseforge.com/minecraft/mc-mods/better-fps-render-distance-fabric
https://www.curseforge.com/minecraft/mc-mods/blur-fabric
https://www.curseforge.com/minecraft/mc-mods/bookshelf
https://www.curseforge.com/minecraft/mc-mods/cachemyenderchest
https://www.curseforge.com/minecraft/mc-mods/camera-utils
https://www.curseforge.com/minecraft/mc-mods/cit-resewn
https://www.curseforge.com/minecraft/mc-mods/cleancut
https://www.curseforge.com/minecraft/mc-mods/client-side-noteblocks
https://www.curseforge.com/minecraft/mc-mods/cloth-config
https://www.curseforge.com/minecraft/mc-mods/connectivity
https://www.curseforge.com/minecraft/mc-mods/continuity
https://www.curseforge.com/minecraft/mc-mods/controlling
https://www.curseforge.com/minecraft/mc-mods/cull-less-leaves
https://www.curseforge.com/minecraft/mc-mods/dynamic-fps
https://www.curseforge.com/minecraft/mc-mods/emoji-type-fabric
https://www.curseforge.com/minecraft/mc-mods/enchantment-descriptions
https://www.curseforge.com/minecraft/mc-mods/entityculling
https://www.curseforge.com/minecraft/mc-mods/essential-mod
https://www.curseforge.com/minecraft/mc-mods/fabric-api
https://www.curseforge.com/minecraft/mc-mods/fabric-language-kotlin
https://www.curseforge.com/minecraft/mc-mods/ferritecore-fabric
https://www.curseforge.com/minecraft/mc-mods/fps-reducer
https://www.curseforge.com/minecraft/mc-mods/free-cam
https://www.curseforge.com/minecraft/mc-mods/gamma-utils
https://www.curseforge.com/minecraft/mc-mods/geckolib
https://www.curseforge.com/minecraft/mc-mods/held-item-info
https://www.curseforge.com/minecraft/mc-mods/indium
https://www.curseforge.com/minecraft/mc-mods/horsestatsvanilla
https://www.curseforge.com/minecraft/mc-mods/i-see-lava
https://www.curseforge.com/minecraft/mc-mods/inventory-hud-forge
https://www.curseforge.com/minecraft/mc-mods/invmove
https://www.curseforge.com/minecraft/mc-mods/invutils
https://www.curseforge.com/minecraft/mc-mods/irisshaders
https://www.curseforge.com/minecraft/mc-mods/item-model-fix
https://www.curseforge.com/minecraft/mc-mods/jade
https://www.curseforge.com/minecraft/mc-mods/journeymap
https://www.curseforge.com/minecraft/mc-mods/jei
https://www.curseforge.com/minecraft/mc-mods/konkrete-fabric
https://www.curseforge.com/minecraft/mc-mods/lambdabettergrass
https://www.curseforge.com/minecraft/mc-mods/lambdynamiclights
https://www.curseforge.com/minecraft/mc-mods/lazydfu
https://www.curseforge.com/minecraft/mc-mods/light-overlay
https://www.curseforge.com/minecraft/mc-mods/limited-chunkloading
https://www.curseforge.com/minecraft/mc-mods/litematica
https://www.curseforge.com/minecraft/mc-mods/lithium
https://www.curseforge.com/minecraft/mc-mods/logical-zoom
https://www.curseforge.com/minecraft/mc-mods/malilib
https://www.curseforge.com/minecraft/mc-mods/mixintrace
https://www.curseforge.com/minecraft/mc-mods/motionblur-fabric
https://www.curseforge.com/minecraft/mc-mods/mouse-tweaks
https://www.curseforge.com/minecraft/mc-mods/no-resource-pack-warnings
https://www.curseforge.com/minecraft/mc-mods/open-loader
https://www.curseforge.com/minecraft/mc-mods/phosphor
https://www.curseforge.com/minecraft/mc-mods/pick-block-pro
https://www.curseforge.com/minecraft/mc-mods/polymorph-fabric
https://www.curseforge.com/minecraft/mc-mods/reacharound
https://www.curseforge.com/minecraft/mc-mods/reeses-sodium-options
https://www.curseforge.com/minecraft/mc-mods/replanter
https://github.com/sebastianrasor/robinhood/releases (disabled)
https://www.curseforge.com/minecraft/mc-mods/smooth-chunk-save
https://www.curseforge.com/minecraft/mc-mods/shulkerboxtooltip
https://www.curseforge.com/minecraft/mc-mods/simple-discord-rich-presence
https://www.curseforge.com/minecraft/mc-mods/sodium
https://www.curseforge.com/minecraft/mc-mods/sodium-extra
https://www.curseforge.com/minecraft/mc-mods/spark
https://www.curseforge.com/minecraft/mc-mods/time-to-live
https://www.curseforge.com/minecraft/mc-mods/tooltipfix
https://modrinth.com/mod/tmp
https://www.curseforge.com/minecraft/mc-mods/torohealth-damage-indicators
https://www.curseforge.com/minecraft/mc-mods/tweakermore
https://www.curseforge.com/minecraft/mc-mods/tweakeroo
https://www.curseforge.com/minecraft/mc-mods/yacl
https://www.curseforge.com/minecraft/mc-mods/yosbr

**Comment by haykam821** [comment]
This seems to be the same issue as #2839. Is the issue fixed by updating Fabric API?
